src/gleam_stdlib.mjs

```diff
@@ -392,11 +392,11 @@ export function regex_scan(regex, string) {
     const submatches = [];
     for (let n = match.length - 1; n > 0; n--) {
       if (match[n]) {
-        submatches[n-1] = new Some(match[n])
-        continue
+        submatches[n - 1] = new Some(match[n]);
+        continue;
       }
-      if(submatches.length > 0) {
-        submatches[n-1] = new None()
+      if (submatches.length > 0) {
+        submatches[n - 1] = new None();
```

Comment:
Ooops. Didn't mean to commit these prettier changes. Should I leave them regardless?

Reply:
Fine to keep them :)